docs/api/rvdss.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Respiratory Virus Detections in Canada
+parent: Data Sources and Signals
+grand_parent: Other Endpoints (COVID-19 and Other Diseases)
+nav_order: 1
+---
+
+# Respiratory Virus Detections in Canada
+
+{: .no_toc}
+
+-   **Source name:** `rvdss`
+-   **Earliest issue available:** Epiweek 36 2013 (2013-09-07)
+-   **Date of last change:** Never
+-   **Available for:** province, lab, region, nation (see [Geography](#geography))
+-   **Time type:** week (see [date format docs](../covidcast_times.md))
+-   **License:** **Open Government Licence - Canada**
+
+This data source of respiratory virus detections in Canada is collected by the [Respiratory Virus Detection Surveillance System](https://health-infobase.canada.ca/respiratory-virus-surveillance/?source=rvdss) (RVDSS) and published by the Public Health Agency of Canada (PHAC). Laboratory tests for various respiratory illnesses from various sentinel laboratories across Canada are reported on a weekly basis year-round to the [Centre for Immunization and Respiratory Infectious Diseases (CIRID)](https://www.canada.ca/en/public-health/services/infectious-diseases/centre-immunization-respiratory-infectious-diseases-cirid.html), Public Health Agency of Canada. The data was originally scraped when weekly online reports, but since June 2024, has been reported on a dynamic dashboard.
+
+NOTE: Pandemic COVID-19 is not reported in this data source.
+
+| Signal                         | Description                                                                                                                                 |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| `sarscov2_tests`               | Number of SARS-CoV-2 laboratory tests <br/> **Earliest Date Available:** 2022-09-03                                                         |
```

Comment:
suggestion: Recommend clarifying in the description that SARS-CoV-2 is the virus that causes COVID-19, both for better searching and because people mostly use the name "COVID-19". I guess maybe it's not technically correct to say that this is a "COVID-19 laboratory test" (?), but we could say that too.

Second, do we know what kind of tests these are? E.g. antigen, nose swab, NAAT (nucleic acid amplification)